Fix GH actions

AmintorDusko opened the pull request:
**Context:**
Some GH actions are failing. This PR will fix that.

**Description of the Change:**

**Benefits:**

**Possible Drawbacks:**

**Related GitHub Issues:**

mlxd commented on .github/workflows/wheel_linux_aarch64.yml:
```diff
@@ -162,29 +162,29 @@ jobs:
 
       - uses: actions/upload-artifact@v3
-        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' }}
+        # if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' }}
```
```suggestion
        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' }}
```

mlxd commented on .github/workflows/wheel_linux_aarch64.yml:
```diff
@@ -162,29 +162,29 @@ jobs:
+    runs-on: ubuntu-latest
 
-    if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
+    # if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
```
```suggestion
    if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
```

mlxd commented on .github/workflows/wheel_linux_aarch64.yml:
```diff
@@ -162,29 +162,29 @@ jobs:
-        arch: [aarch64]
-        pl_backend: ["lightning_qubit"]
-    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+          arch: [aarch64]
+          pl_backend: ["lightning_qubit"]
```
Is there a need to mix 2 and 4 line indentation here?

AmintorDusko replied:
Locally they look the same to me. I will try to fix it.

mlxd commented on .github/workflows/wheel_linux_ppc64le.yml:
```diff
@@ -154,29 +154,29 @@ jobs:
 
       - uses: actions/upload-artifact@v3
-        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' || steps.rc_build.outputs.match != ''}}
+        # if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' || steps.rc_build.outputs.match != ''}}
```
```suggestion
        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' || steps.rc_build.outputs.match != ''}}
```

mlxd commented on .github/workflows/wheel_linux_ppc64le.yml:
```diff
@@ -154,29 +154,29 @@ jobs:
-        arch: [ppc64le]
-        pl_backend: ["lightning_qubit"]
-    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+          arch: [ppc64le]
+          pl_backend: ["lightning_qubit"]
```
Same as earlier re: 2 and 4 line indent

AmintorDusko replied:
Same as earlier.

mlxd commented on .github/workflows/wheel_linux_ppc64le.yml:
```diff
@@ -154,29 +154,29 @@ jobs:
+    runs-on: ubuntu-latest
 
-    if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
+    # if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
```
```suggestion
    if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
```

mlxd commented on .github/workflows/wheel_linux_x86_64.yml:
```diff
@@ -180,7 +180,7 @@ jobs:
 
       - uses: actions/upload-artifact@v3
-        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' || steps.rc_build.outputs.match != ''}}
+        # if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' || steps.rc_build.outputs.match != ''}}
```
```suggestion
        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' || steps.rc_build.outputs.match != ''}}
```

mlxd commented on .github/workflows/wheel_linux_x86_64.yml:
```diff
@@ -189,21 +189,20 @@ jobs:
+    runs-on: ubuntu-latest
 
-    if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
+    # if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
```
```suggestion
    if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
```

mlxd commented on .github/workflows/wheel_macos_arm64.yml:
```diff
@@ -112,7 +112,7 @@ jobs:
 
       - uses: actions/upload-artifact@v3
-        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' }}
+        # if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' }}
```
```suggestion
        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' }}
```

mlxd commented on .github/workflows/wheel_macos_arm64.yml:
```diff
@@ -121,21 +121,20 @@ jobs:
+    runs-on: ubuntu-latest
 
-    if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
+    # if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
```
```suggestion
    if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
```

mlxd commented on .github/workflows/wheel_macos_x86_64.yml:
```diff
@@ -170,7 +170,7 @@ jobs:
 
       - uses: actions/upload-artifact@v3
-        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' || steps.rc_build.outputs.match != ''}}
+        # if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' || steps.rc_build.outputs.match != ''}}
```
```suggestion
        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' || steps.rc_build.outputs.match != ''}}
```

mlxd commented on .github/workflows/wheel_macos_x86_64.yml:
```diff
@@ -179,21 +179,20 @@ jobs:
+    runs-on: ubuntu-latest
 
-    if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
+    # if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
```
```suggestion
    if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
```

mlxd commented on .github/workflows/wheel_noarch.yml:
```diff
@@ -59,7 +59,7 @@ jobs:
 
       - uses: actions/upload-artifact@v3
-        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' }}
+        # if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' }}
```
```suggestion
        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' }}
```

mlxd commented on .github/workflows/wheel_noarch.yml:
```diff
@@ -69,9 +69,9 @@ jobs:
+    runs-on: ubuntu-latest
 
-    if: ${{ github.event_name == 'release' }}
+    # if: ${{ github.event_name == 'release' }}
```
```suggestion
    if: ${{ github.event_name == 'release' }}
```

mlxd commented on .github/workflows/wheel_win_x86_64.yml:
```diff
@@ -185,21 +185,20 @@ jobs:
+    runs-on: ubuntu-latest
 
-    if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
+    # if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
```
```suggestion
    if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
```

mlxd commented on .github/workflows/wheel_win_x86_64.yml:
```diff
@@ -176,7 +176,7 @@ jobs:
 
       - uses: actions/upload-artifact@v3
-        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' || steps.rc_build.outputs.match != ''}}
+        # if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' || steps.rc_build.outputs.match != ''}}
```
```suggestion
        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' || steps.rc_build.outputs.match != ''}}
```

mlxd commented on pennylane_lightning/core/_version.py:
```diff
@@ -16,4 +16,4 @@
 """
 
-__version__ = "0.32.0-dev6"
+__version__ = "0.32.0-dev7"
```
```suggestion
__version__ = "0.32.0-dev8"
```

Since dev7 is taken by the PR, let's update this to there are no conflicts on merging to master.

AmintorDusko replied:
Yes, I know.

AmintorDusko commented:
Since we can observe that the `upload-pypi` actions are working just fine, I removed the testing.
Let me know your thoughts.

vincentmr commented:
> I removed the testing.

I'm not sure I can see those mods? Do you mean `CIBW_TEST_COMMAND` and related steps?

AmintorDusko commented:
> > I removed the testing.
> 
> I'm not sure I can see those mods? Do you mean `CIBW_TEST_COMMAND` and related steps?

What I meant to say was that I removed the testing regarding the upload-pypi.
I removed comments to some lines that were enabling the upload to Pypi.

vincentmr reviewed (approved):
In essence, we can't use `matrix.os` because of the OS version, right? Looks good to me, thanks @AmintorDusko

AmintorDusko commented:
> In essence, we can't use `matrix.os` because of the OS version, right? Looks good to me, thanks @AmintorDusko

Partially yes, but more than this, `matrix.os` writes the OS name in a  different capitalization scheme.
